framework/src/modules/token/cc_commands/cc_forward.ts

```diff
@@ -116,7 +115,7 @@ export class CCForwardCommand extends BaseCCCommand {
 
 		escrowData.amount -= params.amount + params.forwardedMessageFee;
 		await escrowStore.set(methodContext, escrowKey, escrowData);
-		const localTokenID = Buffer.concat([CHAIN_ID_ALIAS_NATIVE, localID]);
+		const localTokenID = Buffer.concat([Buffer.alloc(0), localID]);
```

Comment:
Under endpoint.ts `CHAIN_ID_ALIAS_NATIVE = Buffer.from([0, 0, 0, 1])` is different from the substitute used here. Which logic decides this ?

Reply:
Concept of `ALIAS` is removed and also `cc_forward` is removed, so it's just temporal replacement until this files are removed / replaced in the new issues